frontend/src/components/regions/UsersAndGroups/utils.js

```diff
@@ -0,0 +1,31 @@
+const PR_WARNING_AGE_THRESHOLD = 1
+const PR_ERROR_AGE_THRESHOLD = 2
+const TILDE_ERROR_AGE_THRESHOLD = 3
+const TILDE_WARNING_AGE_THRESHOLD = 2
+const TILDE_DEFAULT_AGE_THRESHOLD = 0
+
+export const timeDifferenceInDays = (time) => {
+  return Math.ceil(Math.abs(new Date() - new Date(time)) / (1000 * 60 * 60 * 24) - 1)
+}
+
+export const getPrStatus = (oldestOpenPrTime) => {
+  const ageInDays = timeDifferenceInDays(oldestOpenPrTime)
+  if (ageInDays >= PR_ERROR_AGE_THRESHOLD){
+    return "error";
+  }
+  if(ageInDays >= PR_WARNING_AGE_THRESHOLD){
+    return "warning"
+  }
+  return "default"
+};
+
+export const getTildeReviewStatus = (oldestOpenTildeReviewTime) => {
+  const ageInDays = timeDifferenceInDays(oldestOpenTildeReviewTime)
+  if (ageInDays >= TILDE_ERROR_AGE_THRESHOLD){
+    return "error";
+  }
+  if(ageInDays <= TILDE_WARNING_AGE_THRESHOLD && ageInDays > TILDE_DEFAULT_AGE_THRESHOLD){
+    return "warning"
+  }
+  return "default"
+};
```

Comment:
Hey Faith could you please store the status strings as constants globally instead to avoid repetition of the strings and re-use the variables instead.